plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -3191,18 +3040,17 @@ def delete_client_role_scope_from_realm(self, payload, clientid, realm="master")
         """
         client_role_scope_url = URL_CLIENT_ROLE_SCOPE_REALM.format(url=self.baseurl, realm=realm, id=clientid)
         try:
-            open_url(client_role_scope_url, method='DELETE', http_agent=self.http_agent, headers=self.restheaders, timeout=self.connection_timeout,
-                     data=json.dumps(payload), validate_certs=self.validate_certs)
+            self._request(client_role_scope_url, method='DELETE', data=json.dumps(payload))
 
         except Exception as e:
-            self.fail_open_url(e, msg='Could not delete roles scope for client %s in realm %s: %s' % (clientid, realm, str(e)))
+            self.fail_request(e, msg='Could not delete roles scope for client %s in realm %s: %s' % (clientid, realm, str(e)))
 
         return self.get_client_role_scope_from_realm(clientid, realm)
 
-    def fail_open_url(self, e, msg, **kwargs):
+    def fail_request(self, e, msg, **kwargs):
```

Comment:
The method `fail_open_url` must be kept (simply add a wrapper that calls `fail_request`) for backwards compatibility. The module utils is part of the public API of the collection.